<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saveTip</class>
 <widget class="QWidget" name="saveTip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:18px;
color:rgb(255,255,255);
font-weight:700;</string>
       </property>
       <property name="text">
        <string>Tip</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/QtTest/icon/close_btn_white.png);
background-color:rgb(0,0,0);
background-position:center;
background-repeat:no-repeat;
border:1px solid;
border-radius:4px;
border-color:rgba(255,255,255,50%);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font-size:14px;
color:rgb(255,255,255);
font-weight:700;</string>
     </property>
     <property name="text">
      <string>聊天记录未保存，退出将丢失记录</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbConfirm">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>56</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>56</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);
font-size:14px;
font-weight: 700;
border-radius:8px;

background-origin: content;
background-position:left;
background-repeat: no-repeat;
text-align:center;
color:#FFFFFF;
border:1px solid;
border-radius:8px;
border-color:#FFFFFF;
</string>
       </property>
       <property name="text">
        <string>Sure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>56</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>56</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);
color:rgb(255,255,255);
font-size:14px;
font-weight: 700;
border-radius:8px;</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
